dts/bindings/serial/raspberrypi,pico-pio-uart.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Ionut Pavel <iocapa@iocapa.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Raspberry Pi Pico PIO UART
+
+compatible: "raspberrypi,pico-pio-uart"
+
+include: [
+  "raspberrypi,pico-pio-device.yaml",
+  uart-controller.yaml,
+  pinctrl-device.yaml
+]
+
+properties:
```

Comment:
uart-controller.yaml now has `stop-bits` and `data-bits`, so you can remove them here

Reply:
Done